packages/backend/src/app/ee/user/enterprise-user-controller.ts

```diff
@@ -0,0 +1,52 @@
+import { FastifyPluginAsyncTypebox, Type } from '@fastify/type-provider-typebox'
+import { ApId, EndpointScope, PrincipalType, assertEqual, assertNotNullOrUndefined } from '@activepieces/shared'
+import { enterpriseUserService } from './enterprise-user-service'
+import { StatusCodes } from 'http-status-codes'
+
+export const enterpriseUserController: FastifyPluginAsyncTypebox = async (app) => {
+    app.get('/', ListUsersRequest, async (req) => {
+        const principalPlatformId = req.principal.platform?.id
+        const requestPlatformId = req.query.platformId
+        assertEqual(principalPlatformId, requestPlatformId, 'principalPlatformId', requestPlatformId)
+
+        return enterpriseUserService.list({
+            platformId: req.query.platformId,
```

Comment:
I wouldn't ask for platform id as its owner of all entities

Reply:
Imagine someone using api key, they would need two calls to figure out platform id then call this endpoint